mittab/templates/outrounds/pretty_pairing.html

```diff
@@ -31,8 +31,22 @@
 		    </tr>
 		    {% for pairing in outround.rounds %}
 			<tr>
-			    <td>{{pairing.gov_team.display}}</td>
-			    <td>{{pairing.opp_team.display}}</td>
+				<td>
+					<div class="teamNames">
+						{{pairing.gov_team.display}}
+					</div>
+					<div class="memberNames" style="display: none;">
+						{{pairing.gov_team.debaters_display}}
+					</div>
+				</td>
+				<td>
+					<div class="teamNames">
+						{{pairing.opp_team.display}}
+					</div>
+					<div class="memberNames" style="display: none;">
+						{{pairing.opp_team.debaters_display}}
+					</div>
```

Comment:
If `debaters_public` is false, we shouldn't put this in the HTML at all, even with `display: none`